x/feesplit/spec/04_transactions.md

```diff
@@ -6,19 +6,19 @@ order: 4
 
 This section defines the `sdk.Msg` concrete types that result in the state transitions defined on the previous section.
```

Comment:
Nit: docs grammar for intro;
This section defines the `sdk.Msg` concrete types that result in the state transitions defined **_in_** the previous section.